apps/judicial-system/api/tsconfig.json

```diff
@@ -6,7 +6,6 @@
     "target": "es2015",
     "strict": true
   },
-  "allowJs": true,
```

Comment:
I deleted it because `allowJs` is part of the `compilerOptions` object and since there is no javascript file in the folder, it's most likely unnecessary

Reply:
👍  Should be ok.